### ISSUE TITLE
Wrong width for some Cyrillic characters

### ISSUE THREAD

**Issue by dveeden** [jquast/wcwidth#13]
The width for these are not correct:
* COMBINING CYRILLIC HUNDRED THOUSANDS SIGN
* COMBINING CYRILLIC HUNDRED MILLIONS SIGN

Test case:
```python
import unicodedata
import math
import wcwidth

charnames = [u'COMBINING CYRILLIC HUNDRED THOUSANDS SIGN',
             u'COMBINING CYRILLIC HUNDRED MILLIONS SIGN',
             u'CIRCLED LATIN CAPITAL LETTER A',
             u'KANGXI RADICAL BIRD']

for charname in charnames:
    print(u'Character: {}'.format(charname))
    c = unicodedata.lookup(charname)
    cwidth = wcwidth.wcwidth(c)
    spacing = int(math.floor(2 - cwidth)) * u' '
    print(u"123456789")
    print(u"123{}{}6789".format(c, spacing))
    print(u"Char width: {}".format(cwidth, spacing))
```
This is printed with Python 2.7 and wcwidth 0.1.4:
```
Character: COMBINING CYRILLIC HUNDRED THOUSANDS SIGN
123456789
123҈ 6789
Char width: 1
Character: COMBINING CYRILLIC HUNDRED MILLIONS SIGN
123456789
123꙱ 6789
Char width: 1
Character: CIRCLED LATIN CAPITAL LETTER A
123456789
123Ⓐ 6789
Char width: 1
Character: KANGXI RADICAL BIRD
123456789
123⿃6789
Char width: 2
```
Related: https://github.com/dbcli/mycli/issues/149

**Comment by jquast** [comment]
Thank you!

**Comment by jquast** [comment]
Note also the https://github.com/jquast/wcwidth/blob/master/bin/wcwidth-browser.py#L12 program distributed only in the github VCS tree which is a general purpose utility performing the same actions

**Comment by jquast** [comment]
This was resolved with the release of wcwidth 0.1.5 https://github.com/jquast/wcwidth/releases/tag/0.1.5

Given your sample program, this is the difference between 0.1.4 and 0.1.5:
```
 Character: COMBINING CYRILLIC HUNDRED THOUSANDS SIGN
 123456789
-123҈ 6789
-Char width: 1
+123҈  6789
+Char width: 0
 Character: COMBINING CYRILLIC HUNDRED MILLIONS SIGN
 123456789
-123꙱ 6789
-Char width: 1
+123꙱  6789
+Char width: 0
 Character: CIRCLED LATIN CAPITAL LETTER A
 123456789
 123Ⓐ 6789
```

Thanks for your time filing detailed report @dveeden !
